{"type":"PR","pr":{"number":89,"head":"315c0f3f48359060f2fc16bc866b4d491df57dfd"},"user":"yuuki.nakamura.0828","status":"draft","branch":"cms/2018-03-13-railsの例外をslackに通知する「exception_notification」","collection":"post","title":"Railsの例外をSlackに通知する「exception_notification」","description":"「exception_notification」を利用して、Railsの例外をSlackに通知します。","objects":{"entry":{"path":"_posts/2018-03-13-railsの例外をslackに通知する「exception_notification」.md","sha":"54d5ed608cc5a32a7450259b8b7c6e5d6c99bbee"},"files":[{"path":"/images/uploads/slack_logo.png","sha":"a80563df384a9cdb75096b379dee1802dbf99660"}]},"timeStamp":"2018-03-13T06:16:03.040Z"}